src/printapp/templates/index.html

```diff
@@ -295,7 +312,7 @@ <h3 class="subheader">Welcome, {{email}}</h3>
       <div class="row">
         <div class="large-2 medium-2 small-5 columns end">
           <label>Copies
-            {{input type="number" value=copies min="1"}}
+            {{input type="number" value=copies min="1" step="1"}}
```

Comment:
Yah, I think you have to change it input type to text, then use a regular expression instead. See the link in the issue.